scene/gui/color_picker.cpp

```diff
@@ -176,14 +176,15 @@ void ColorPicker::_update_color() {
 	updating = true;
 
 	for (int i = 0; i < 4; i++) {
-		scroll[i]->set_step(0.01);
 		if (raw_mode_enabled) {
+			scroll[i]->set_step(0.01);
 			scroll[i]->set_max(100);
 			if (i == 3)
 				scroll[i]->set_max(1);
 			scroll[i]->set_value(color.components[i]);
 		} else {
-			const int byte_value = color.components[i] * 255;
+			scroll[i]->set_step(1);
```

Comment:
For the reference, this had been changed in #18981 and then reverted in #18988, but I think the problematic change back then was forcing a max of `1` for the raw mode. Changing the step as done here should be fine.

Reply:
BTW we could likely set the raw mode max to `1.0` and enable `set_allow_greater(true)` now that the feature is available. But that's probably more stuff for after 3.1.